rxandroid/src/main/AndroidManifest.xml

```diff
@@ -12,4 +12,4 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<manifest package="rx.android" />
\ No newline at end of file
+<manifest package="io.reactivex.android" />
```

Comment:
Right right. Still looks strange to make. I'd prefer 1 to 1 mapping of the root package name of classes and group name of the artifact because it's just more obvious for users